python/ray/tests/test_component_failures.py

```diff
@@ -311,8 +312,12 @@ def check_components_alive(cluster, component_type, check_component_alive):
 @pytest.mark.parametrize(
     "ray_start_cluster", [{
         "num_cpus": 8,
-        "num_nodes": 4
-    }], indirect=True)
+        "num_nodes": 4,
+        "_internal_config": json.dumps({
+            "num_heartbeats_timeout": 100
```

Comment:
We need to increase heartbeat timeout for this test, because submitting tasks is much faster than before. This causes a large amount of requests (~10k) being sent to Redis suddenly, and delays the time when the raylet monitor can receive the heartbeat.